<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeeduLoginWidget</class>
 <widget class="QWidget" name="WeeduLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#WeeduLoginWidget
{
background-color: #999999;
border-radius:2px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetBack" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widgetBack
{
background-color: #999999;
border-radius:8px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widgetTop" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget#widgetTop
{
background-color: rgb(255, 255, 255);
border-radius:0px;
}</string>
        </property>
        <widget class="QPushButton" name="pushButtonClose">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>10</y>
           <width>15</width>
           <height>15</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border-image: url(:/images/widget_clos_1.png);
background-repeat:no-repeat;
border:1px solid #d8d8d8;
border:none;
}
QPushButton:hover
{
border-image:url(:/images/widget_clos_2.png);
}
QPushButton:pressed
{
border-image:url(:/images/widget_clos_2.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditAccount">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>251</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom:1px solid #d7d7d7;
border-radius:5px;
color:#666666;
text-align:left;</string>
         </property>
         <property name="text">
          <string>mingquan</string>
         </property>
         <property name="placeholderText">
          <string>输入号码</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>251</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-bottom:1px solid #d7d7d7;
border-radius:5px;
/*font-family:PingFangSC-Regular;
font-size:14px;*/
color:#666666;
text-align:left;</string>
         </property>
         <property name="text">
          <string>a12345678</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>输入密码</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonLogin">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>220</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border-image: url(:/images/btn_unLogin.png);
	/*background:#6ed56c;
	border-radius:8px;
	font-family:PingFangSC-Regular;
	font-size:18px;*/
	color:#ffffff;
}
/*QPushButton:hover{
    background-color:#6ee56c
}
QPushButton:pressed{
    background-color:#6ee56c
}*/

</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="labelLogin">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>50</y>
           <width>102</width>
           <height>64</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>19</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">/*font-family:Arial;
font-size:36px;
color:#666666;
text-align:left;*/

border-image: url(:/images/logo.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>300</y>
           <width>12</width>
           <height>8</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border-image: url(:/images/btn_offControl.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetBottom" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>76</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#widgetBottom
{
background-color: #f4f9fb;
}</string>
        </property>
        <widget class="QPushButton" name="pushButtonRegister">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>50</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
color:#999999;
/*text-align:right;*/
border-radius:2px;
}
QPushButton:hover
{
color:#666666;
}
</string>
         </property>
         <property name="text">
          <string>注册账号</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxRemember">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>20</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox
{
color:#999999;
text-align:left;
}

QCheckBox::indicator{  
    width: 16px;  
    height: 16px;  
} 

/*QCheckBox::indicator:unchecked
{  
    image: url(:/images/unChockBox.png);
}  

QCheckBox::indicator:checked
{  
    image: url(:/images/chockBox.png);
}  
*/
QCheckBox:hover
{
color:#666666;
}
</string>
         </property>
         <property name="text">
          <string>记住密码</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxAutoLogin">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>50</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox
{
color:#999999;
text-align:left;
}

QCheckBox::indicator{  
    width: 16px;  
    height: 16px;  
} 

/*QCheckBox::indicator:unchecked
{  
    image: url(:/images/unChockBox.png);
}  

QCheckBox::indicator:checked
{  
    image: url(:/images/chockBox.png);
}  
*/
QCheckBox:hover
{
color:#666666;
}
</string>
         </property>
         <property name="text">
          <string>自动登录</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonForget">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
color:#999999;
/*text-align:right;*/
border-radius:2px;
}
QPushButton:hover
{
color:#666666;
}
</string>
         </property>
         <property name="text">
          <string>忘记密码</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
